core/src/main/kotlin/com/malinskiy/marathon/execution/json/FileSerializer.kt

```diff
@@ -0,0 +1,14 @@
+package com.malinskiy.marathon.execution.json
```

Comment:
The package differs from [upstream](https://github.com/MarathonLabs/marathon/commit/9b2009913dfe381805ee6fb2be0f2affc1387ee7)